scripts/generate-merkle-root.ts

```diff
@@ -1,18 +1,72 @@
 import { program } from 'commander'
 import fs from 'fs'
 import { parseBalanceMap } from '../src/parse-balance-map'
+import { keccak256 } from 'ethereumjs-util'
+import { utils } from 'ethers'
+
+const sfc32 = (a: number, b: number, c: number, d: number) => () => {
+  a = a | 0
+  b = b | 0
+  c = c | 0
+  d = d | 0
+  const t = (a + b | 0) + d | 0
+  d = d + 1 | 0
+  a = b ^ b >>> 9
+  b = c + (c << 3) | 0
+  c = (c << 21) | (c >>> 11)
+  c = c + t | 0
+  return (t >>> 0) / 4294967296
+}
 
 program
   .version('0.0.0')
   .requiredOption(
     '-i, --input <path>',
-    'input JSON file location containing a map of account addresses to string balances'
+    'input JSON file location containing a map of account addresses to string balances')
+  .requiredOption(
+    '-s, --seed <number>',
+    'You must sepcify a random seed!'
+  )
+  .requiredOption(
+    '-n, --num <number>',
+    'You must sepcify the number random addresses!'
   )
 
 program.parse(process.argv)
 
-const json = JSON.parse(fs.readFileSync(program.input, { encoding: 'utf8' }))
+const scalingFactor1: number = 0xffffffffffffffffff; // this is a multiplier to shift the decimal part and get a large integer number
```

Comment:
Not exactly a problem, but I found this [reference](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number):
> Therefore, the mantissa's precision is 2-52 (obtainable via [Number.EPSILON](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/EPSILON)), or about 15 to 17 decimal places; arithmetic above that level of precision is subject to [rounding](https://en.wikipedia.org/wiki/Floating-point_arithmetic#Representable_numbers,_conversion_and_rounding).

This means that `sfc32` would never generate a value with more than 17 decimals, meaning that `10**18` would be a sufficient scaling factor.

However, the choice doesn't change the accuracy of the generation method, so no action is required.